storage_service/locations/models/package.py

```diff
@@ -91,6 +91,10 @@ class Package(models.Model):
     misc_attributes = jsonfield.JSONField(blank=True, null=True, default={},
         help_text='For storing flexible, often Space-specific, attributes')
 
+    # Provide access, without join, to last fixity check result/date
+    last_fixity_check_result = models.NullBooleanField()
+    last_fixity_check_date = models.DateTimeField(blank=True, null=True)
```

Comment:
Are these duplicates of `FixityLog.[success|time_reported]`?  I'm concerned about them getting out of sync.  Maybe make it a foreign key and look at this performance tweak if it becomes a problem?  Are there case where we need this info but no FixityLog object exists?